src/service/action-impl.js

```diff
@@ -71,16 +75,19 @@ export class ActionInvocation {
    * @param {!Node} target
    * @param {string} method
    * @param {?JSONType} args
+   * @param {?string} argsExpression
    * @param {?Element} source
    * @param {?Event} event
    */
-  constructor(target, method, args, source, event) {
+  constructor(target, method, args, argsExpression, source, event) {
```

Comment:
I don't believe `object-string` is a possible key. How about that? If we do use that, we should absolutely add a test that proves the parser will throw trying to use it outside of this object literal codepath.